openmdao/components/submodel_comp.py

```diff
@@ -0,0 +1,447 @@
+"""Define the SubmodelComp class for evaluating OpenMDAO systems within components."""
+
+from openmdao.core.explicitcomponent import ExplicitComponent
+from openmdao.utils.general_utils import find_matches
+from openmdao.core.constants import _SetupStatus, INF_BOUND
+
+
+class SubmodelComp(ExplicitComponent):
+    """
+    System level container for systems.
+
+    Parameters
+    ----------
+    model : <System>
```

Comment:
model has to be a Group now, so should replace <System> with <Group>.

Reply:
changed `<System>` to `<Group>`